web/v1/redshift/CHANGELOG

```diff
@@ -0,0 +1,12 @@
+Version 1.2.0 (2021-01-25)
```

Comment:
Can we please also add Version 1.1.0 - 5th November, referencing only the below commit?

https://github.com/snowplow/data-models/commit/45d2b96e9b266f5addbc37f6a7df891b8f392f73